kernal/drivers/x16/sprites.s

```diff
@@ -183,9 +187,15 @@ sprite_set_position:
 	sta VERA_DATA0 ; offset 4: Y lo
 	lda r1H
 	sta VERA_DATA0 ; offset 5: Y hi
-	lda #3 << 2
-	sta VERA_DATA0 ; offset 6: set zdepth to 3
-
+	ldx #(^VERA_SPRITES_BASE)
+	stx VERA_ADDR_H
+	lda VERA_DATA0
+	and #%00001100
```

Comment:
So this check whether to set the bits costs us 10 cycles (11 if we cross a page boundary here), but just setting the bits anyways would cost 10 cycles. Why do the test?

Reply:
In #71 the complaint was that moving the mouse cursor stomps on the flip and Z-depth bits.  If they have a z-depth set other than 3, the request is that we should preserve it.  However, if it's not set at all (invisible sprite), then we do need to set it.

Reply:
Ah, yes, I see it now. :P